homeassistant/components/switchbot/sensor.py

```diff
@@ -75,6 +75,12 @@
         state_class=SensorStateClass.MEASUREMENT,
         device_class=SensorDeviceClass.POWER,
     ),
+    "attempt_state": SensorEntityDescription(
+        key="attempt_state",
+        translation_key="attempt_state",
+        native_unit_of_measurement="",
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
```

Comment:
what's this? What can the values be?

Reply:
From what I can tell it tracks _something_ that changes when a successful attempt was made, it increases by 2 on a successful attempt. I haven't had the time to check yet, but is there any way to change the type from a normal sensor to something like a binary sensor to make it easier to use in automations?